File: SubsifyFrontend/Util/Http/ApiResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace SubsifyFrontend.Util.Http
{
    public class ApiResponse
    {
        public int code { get; set; }
        public string message { get; set; }
        public List<object> data { get; set; }

        public ApiResponse(int code) { this.code = code; }
    }
}

File: mobile/app/src/main/java/com/example/subsify/Util/ApiResponse.java
package com.example.subsify.Util;
import java.util.List;

public class ApiResponse {
    private int code;
    private String message;
    private List<Object> data;

    public int getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }

    public List<Object> getData() {
        return data;
    }
}
